components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -217,19 +219,26 @@ class NewTabPage extends React.Component<Props, State> {
   }
 
   trackCachedImage () {
+    console.debug('trackCachedImage')
     if (this.state.backgroundHasLoaded) {
+      console.debug('Resetting to new image')
       this.setState({ backgroundHasLoaded: false })
     }
     if (this.imageSource) {
       const imgCache = new Image()
+      // Store Image in class so it doesn't go out of scope
+      this.imgCache = imgCache
```

Comment:
Wow, does this fix it? That's gnarly :laughing:

Reply:
Yep. I had a feeling after I added logging to onload (and onerror) and just was never getting the event called. So I assume it was just garbage collected, or perhaps an optimization on Image element fetching to cancel if it's out of scope entirely. Not too sure which one, but yes it seems to fix it!